README.adoc

```diff
@@ -4,19 +4,22 @@ A FaaS for Kubernetes
 
 == Installation of latest release
 
-You can install the latest release using link:Getting-Started.adoc[a Helm chart]. 
+You can install the latest (v0.0.3) release using link:Getting-Started.adoc[a Helm chart].
 
-== Installation of the riff CLI
+The https://github.com/projectriff/riff/tree/v0.0.3/samples[Tag v0.0.3 samples] will work together
+with the https://github.com/projectriff/riff/releases[v0.0.3 CLI].
```

Comment:
I think the link should be https://github.com/projectriff/riff/releases/tag/v0.0.3